File: java/PML/src/tags/TagText.java
package tags;

public class TagText extends Tag{
	private String characterData;
	
	public void setCharacterData(String characterData) {
		this.characterData = characterData;
	}
	
	@Override
	public void output() {
		System.out.println(this.characterData);
	}
}

File: php/classes/PML/Tags/TagText.php
<?php namespace PML\Tags;

class TagText extends Tag {

    private $characterData = '';

    /**
     * Construct the base type, Tag
     *
     * @param Array $attributes
     * @param Array $childTags
     */
    public function __construct($characterData)
    {
        $this->characterData = $characterData;
    }

    /**
     * Define some output for this tag
     */
    public function output()
    {
        echo $this->characterData;
    }
}
